module/documents/actor/actor.mjs

```diff
@@ -673,6 +673,7 @@ export default class Actor5e extends SystemDocumentMixin(Actor) {
     }
 
     hp.value = Math.min(hp.value, hp.max + (hp.tempmax ?? 0));
+    hp.damage = hp.max + (hp.tempmax ?? 0) - hp.value;
```

Comment:
Is `hp.temp` left out on purpose?

Reply:
Well I suppose that depends on what you see this damage number as. The way it is currently being used in `applyDamage` it needs to know how many HP the player's current hit points are below their maximum for calculating healing.

`15 (max) - 10 (current) = 5 (damage able to be healed)`
`15 (max) + 2 (temp max) - 10 (current) = 7 (damage able to be healed)`

Temp HP is something that is sort of orthogonal to this. While it can absorb damage, it shouldn't affect healing in any way.

`15 (max) - 10 (current) - 5 (temp) = 0 (you're totally fine! nothing to heal here!)`